extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -238,18 +238,13 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 		}
 	});
 
-	appContext.apiWrapper.registerCommand('azure.resource.openInAzurePortal', async (node?: TreeNode) => {
-		if (!node) {
-			return;
-		}
-
-		const treeItem: azdata.TreeItem = await node.getTreeItem();
-		if (!treeItem.payload) {
-			return;
-		}
-		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: true });
+	appContext.apiWrapper.registerCommand('azure.resource.openInAzurePortal', async (connectionProfile: azdata.IConnectionProfile) => {
 
-		if (!connectionProfile.azureResourceId) {
+		if (
```

Comment:
Do you want to pop up an error in this case? The user wouldn't know what happened to the click otherwise

Reply:
Yeah I can put an error there, the error shouldn't happen though because the menu option only shows up if those values are there.